harness/determined/deploy/local/mount.yaml

```
@@ -3,4 +3,4 @@ version: "3.7"
 services:
   determined-master:
     volumes:
-      -  ${DET_MASTER_CONFIG:-/usr/local/determined/etc/master.yaml}:/etc/determined/master.yaml
+      -  ${DET_MASTER_CONFIG:-/etc/master.yaml}:/etc/determined/master.yaml
```

```suggestion
      -  ${DET_MASTER_CONFIG}:/etc/determined/master.yaml
```

first I was going to suggest that this variable should default to `/etc/determined/master.yaml`, but looking at the code, it seems this mount.yaml is not used unless DET_MASTER_CONFIG is set to something real. so maybe this default is not a thing at all?

---

hey, that's a good point.  That makes one less breaking change than I was thinking.

What do you think about automatically mounting `/etc/master.yaml` when it exists?  That seems like reasonable behavior to me...

---

it doesn't really sound appealing to me.

1. this behavior didn't exist before, so using `/etc/determined/master.yaml` as a path probably makes more sense.
2. "native" determined master from a linux package running on the host will use the same config file.
3. which means if you run native and then try `det deploy local`, you'll suddenly conflate their configs and run into expected side effects.